document/CoreFormat.mdk

```diff
@@ -528,7 +586,7 @@ The `#pragma` warning construct in C# is such a syntactic construct.
 
 ### `fixes` property { #result-fixes }
 
-A `result` object may contain a property names `fixes`, whose value is a JSON array of `fix` objects (&sect;[#fix]).
+A `result` object may contain a property names `fixes`, whose value is a JSON array of zero or more `fix` objects (&sect;[#fix]).
```

Comment:
@michaelcfanning The only other place in the spec where we have an array of "zero or more" somethings is the `runLog.results` property, which is indeed mandatory.

Yes, having an array be optional is a slight inconvenience for consumers, who have to check for null before operating on the array.

OK, I'll change the property to mandatory ("shall").